aspnetcore/signalr/hubs.md

```diff
@@ -120,6 +120,56 @@ By default, a server hub method name is the name of the .NET method. To change t
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/signalr/hubs/samples/6.x/SignalRHubsSample/Snippets/Hubs/ChatHub.cs" id="snippet_HubMethodName" highlight="1":::
 
+## Inject services into your hub
+
+> [!NOTE]
+> The following feature is only available in .NET 7.0+
```

Comment:
Did this instead of duplicating everything and using a moniker range. AFAIK there is not way to nest moniker ranges.

Reply:
Correct ... they can't be nested. I recommend versioning the whole topic now and signaling Rick to activate the version selector for the whole doc set. Place the new versioned content at the **end of the article tho**. Otherwise, it will break bookmark links during preview.